### PR TITLE
disable publishing docker image to Docker Hub

### PR DESCRIPTION
**Pull request by folti**

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build FAILURE

**Comment by kira-syslogng** [comment]
Build FAILURE

**Comment by folti** [comment]
@kira-syslogng please rebuild

**Comment by folti** [comment]
@kira-syslogng test this please;

**Comment by kira-syslogng** [comment]
Build FAILURE

File: lib/ivykis
Comment (kovgeri01) [inline lib/ivykis:1]: You accidentally updated the `ivykis` submodule which causes the tests to fail can you revert this?

**Review by kovgeri01 — CHANGES_REQUESTED** [review]

File: lib/ivykis
[the remark this answers is not in the document]
Comment (folti) [reply lib/ivykis:1]: done

**Review by HofiOne — APPROVED** [review]

**Review by kovgeri01 — APPROVED** [review]

**Comment by HofiOne** [comment]
Credentials are fixed, publish is fine, closing.

**Pull request closed without merging**